Typescript: Properties 'limit' and 'skip' do not exist on type Aggregate

<!-- *Before creating an issue please make sure you are using the latest version of mongoose -->

**Do you want to request a *feature* or report a *bug*?**
I'd like to report a bug

**What is the current behavior?**
I updated from Mongoose 5.10.19 with @types/mongoose to 5.11.6 and got a new error from typescript: 
`error TS2339: Property 'limit' does not exist on type 'Aggregate<any[]>'`
on a line like this:
`const data = await Model.aggregate(aggregationPipeline).skip(skip).limit(limit);`

**If the current behavior is a bug, please provide the steps to reproduce.**
<!-- If you can, provide a standalone script / gist to reproduce your issue -->
Compiling the following with typescript should reproduce the issue:
```
import mongoose from 'mongoose';

mongoose.connect('mongodb://localhost:27017/test', {
  useNewUrlParser: true,
  useCreateIndex: true,
  useUnifiedTopology: true,
});

const personSchema = new mongoose.Schema({name: String})
const personModel = mongoose.model('Person', personSchema);

const doSomething = async () => {
  const data = await personModel.aggregate().skip(10).limit(10);
  console.log(data);
}

doSomething();
```

**What is the expected behavior?**
The program should compile as before

**What are the versions of Node.js, Mongoose and MongoDB you are using? Note that "latest" is not a version.**
NodeJs: 14.11.0
Mongoose: 5.11.6
MongoDB: 4.2.9

<!-- You can print `mongoose.version` to get your current version of Mongoose: https://mongoosejs.com/docs/api.html#mongoose_Mongoose-version -->

--- a/index.d.ts
+++ b/index.d.ts
@@ -2236,6 +2236,12 @@ declare module "mongoose" {
     /** Sets the hint option for the aggregation query (ignored for < 3.6.0) */
     hint(value: object | string): this;
 
+    /**
+     * Appends a new $limit operator to this aggregate pipeline.
+     * @param num maximum number of records to pass to the next stage
+     */
+    limit(num: number): this;
+                                          
     /** Appends new custom $lookup operator to this aggregate pipeline. */
     lookup(options: any): this;
 
@@ -2265,6 +2271,12 @@ declare module "mongoose" {
 
     /** Sets the session for this aggregation. Useful for [transactions](/docs/transactions.html). */
     session(session: mongodb.ClientSession | null): this;
+    
+    /**
+     * Appends a new $skip operator to this aggregate pipeline.
+     * @param num number of records to skip before next stage
+     */
+    skip(num: number): this;
 
     /** Appends a new $sort operator to this aggregate pipeline. */
     sort(arg: any): this;

